source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -97,6 +108,22 @@ content: |
            
            BsonBinary dataKeyId = clientEncryption.createDataKey("aws", dataKeyOptions);
            String base64DataKeyId = Base64.getEncoder().encodeToString(dataKeyId.getData());
+     .. tab::
+        :tabid: nodejs
+
+        .. code-block:: javascript
+        
+           const encryption = new ClientEncryption(client, {
+               keyVaultNamespace,
+               kmsProviders
+           });
+           encryption.createDataKey('aws', {
+               masterKey: { 
+                 key: <Master Key ARN>,// e.g."arn:aws:kms:us-east-2:111122223333:alias/test-key"
+                  region: 'us-east-1' 
+               },
+               keyAltNames: ['key1']
```

Comment:
Did we ever explain `keyAltNames`? If not, is it something we should include in the guide? Otherwise, should we leave it out of the code examples?

Reply:
You're right we don't need keyAltNames, so I've removed it per your suggestion and as to not confuse users. 

From the mongo csfle npm package README (https://www.npmjs.com/package/mongodb-client-encryption): (keyAltName is) "An optional list of string alternate names used to reference a key. If a key is created with alternate names, then encryption may refer to the key by the unique alternate name instead of by _id."